specs/core/0_beacon-chain.md

```diff
@@ -239,17 +239,17 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_DEPOSITS` | `2**4` (= 16) |
 | `MAX_VOLUNTARY_EXITS` | `2**4` (= 16) |
 
-### Domain types
+### Tag types
```

Comment:
Maybe we should call these "Signature Tags" or something a little bit more descriptive in the header.

If you want to leave the name as-is, then let's at least put a few words in this section to describe what we use them for.